seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -265,8 +265,8 @@
       "description": [
         "Convert a date range consisting of two dates formatted as <code>YYYY-MM-DD</code> into a more readable format.",
         "The friendly display should use month names instead of numbers and ordinal dates instead of cardinal (<code>1st</code> instead of <code>1</code>).",
-        "Do not display information that is redundant or that can be inferred by the user: if the date range ends in less than a year from when it begins, do not display the ending year. If the range ends in the same month that it begins, do not display the ending year or month.",
-        "Additionally, if the date range begins in the current year and ends within one year, the year should not be displayed at the beginning of the friendly range.",
+        "Do not display information that is redundant or that can be inferred by the user: if the date range ends in less than a year from when it begins, do not display the <em>ending year<em>. If the range ends in the same month that it begins, do not display the <em>ending year or month<em>.",
+        "Additionally, if the date range begins in the <em>current year<em> and ends within one year, the year should not be displayed at the <em>beginning<em> of the friendly range.",
```

Comment:
All of the `<em>` tags need to be closed with `</em>` tags instead of another `<em>` tag.